src/Microsoft.FeatureManagement/Microsoft.FeatureManagement.csproj

```diff
@@ -15,7 +15,7 @@
     <SignAssembly>true</SignAssembly>
     <DelaySign>false</DelaySign>
     <AssemblyOriginatorKeyFile>..\..\build\Microsoft.FeatureManagement.snk</AssemblyOriginatorKeyFile>
-    <LangVersion>9.0</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
I feel it would be valuable to have a comment explaining why we specify v8 and not a later version.